pkg/app/couchbase.go

```diff
@@ -49,15 +49,16 @@ type CouchbaseDB struct {
 	clusterChart  helm.ChartInfo
 }
 
+// Last tested woking version "0.1.2"
 func NewCouchbaseDB(name string) App {
 	return &CouchbaseDB{
 		name: name,
 		operatorChart: helm.ChartInfo{
 			Release:  appendRandString(fmt.Sprintf("%s-operator", name)),
 			RepoName: helm.CouchbaseRepoName,
 			RepoURL:  helm.CouchbaseRepoURL,
-			Chart:    "couchbase-operator",
 			Version:  "0.1.2",
```

Comment:
do we want to remove this version from here?

Reply:
Installation of the latest version of couchbase helm chart itself is failing. Removing version will cause the test to fail.